src/main/java/com/ita/if103java/ims/controller/WarehouseController.java

```diff
@@ -17,11 +19,13 @@
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
+import springfox.documentation.annotations.ApiIgnore;
 
 import java.util.List;
 import java.util.Map;
 
 @RestController
+@CrossOrigin("http://localhost:4200/")
```

Comment:
So common configs is not working, @MarrMika ?

Reply:
Work!